### DISCUSSION TITLE
Actors go invisible

### DISCUSSION THREAD

**Discussion by AnimeFloof** [ErinaSugino/Starbound-SBR-Mod#119] [Trouble Shooting] [answered]
Like the title states, actors just straight up go invisible, myself included. If I try to shoot em with Aphrodite's Bow, they go poof. I can still interact, but then I go poof and can't see anything!

[starbound.log](https://github.com/user-attachments/files/15775551/starbound.log)

**Comment by ErinaSugino** [comment]
https://github.com/ErinaSugino/Starbound-SBR-Mod/wiki/Advanced-Troubleshooting

Also, for future reference - please attach log files as actual file, not copy'n'paste the entire bible in text form. This makes it very hard to navigate the discussion.

**Reply by AnimeFloof** [reply]
Sorry! First time doing this. I can just edit it rq!

**Reply by AnimeFloof** [reply]
Oh I guess you sending that link was basically telling me to go re-read.

I don't have anything that it states would cause an issue. I've never used Naturally Horny or the Extended Dialogue. I've used Alluring Toys, but I made sure to remove that from the mods folder. Same goes for terramon extensions

**Reply by ErinaSugino** [reply, marked as answer]
You do have Teratophilia Scenes though, and the wrong version at that.

**Reply by AnimeFloof** [reply]
Ah I see it now. I don't know why the creator has two different versions under two different names. It confused me and I thought it was just for something else. Goes to show I really didn't read. My bad. Thank you for humoring this though, sorry if this was an annoyance at all.

**Reply by ErinaSugino** [reply]
It's just like how POV Redux has two versions - 4.1.x (for OG SxB) and 4.2.0 (for SBR). Just, way less conveniently named. Moving forward, people hopefull finally stop using old Sexbound so there is no need for multiple versions - but then again, getting old, long since abandoned mods to become SBR compatible is already a tall ask to begin with.

**Comment by akisephila** [comment]
Hi, having a similar issue where for some reason it can't seem to load assets correctly, resulting in... this.
![image](https://github.com/ErinaSugino/Starbound-SBR-Mod/assets/77479717/4f0262d6-959b-4132-bb63-eb779541f219)
Here's the log file
[starbound.log](https://github.com/user-attachments/files/15817845/starbound.log)
Here's a screenshot of my mods folder. I'm fairly certain I installed the correct versions of everything.
![image](https://github.com/ErinaSugino/Starbound-SBR-Mod/assets/77479717/2b8e61f5-86c9-45fa-9412-52067f0dcc66)
I did also install [this](https://www.loverslab.com/files/file/26677-sxb-vanilla-races-animation-retexture-as-anime-style/) mod as detailed in its instructions, but I was having this issue before installing that anyway.

**Reply by akisephila** [reply]
I also enabled the info and debug config options before posting the log

**Reply by akisephila** [reply]
Additionally, using Contraceptive Patches to become a Futanari only offers Facesitting position with same-sex actors. Facesitting position lacks the entire body, not just the head.

**Reply by ErinaSugino** [reply]
Wow, that's a lot to unpack.
First: "Contraceptive Patch" changes no gender. It makes you not ableto get pregnant. So naturally, no change in position availability is given.
Second: Something is really borked for you there. Starbound screams occasionally that it I guess frame declarations are missing? The missing head, however, is a full mystery as the it uses the vanilla species head sprite and nothing ever errors there.

**Reply by ErinaSugino** [reply]
Ok, so, update. Still no clue what's wrong with the head. Though the frame errors specifically for positions "facesitting" and "reverse_cowgirl", that make the entire body disappear, are caused by using POV Redux in combination with the Pygophilia POV modset. They faultily override eachother. You shouldn't be needing POV Redux for Pygophilia POV, as it seems to be it's own POV implementation.

**Reply by akisephila** [reply]
ah ok, thanks.

**Reply by akisephila** [reply]
Is it possible one of the steam workshop mods i've subscribed to could also be causing incompatibility/loading issues? if so, I'll put them all into a collection to make it easier to see whats causing issues.

**Reply by akisephila** [reply]
> Is it possible one of the steam workshop mods i've subscribed to could also be causing incompatibility/loading issues? if so, I'll put them all into a collection to make it easier to see whats causing issues.

https://steamcommunity.com/sharedfiles/filedetails/?id=3267400460 here

**Reply by akisephila** [reply]
also, to amend an error; i was not using the contraceptive patch, i was using the _hormone_ patch

**Reply by akisephila** [reply]
Ok, nevermind. After removing the Pygophilia mods, literally everything works correctly. This is what you get when a mod provides literally zero installation instructions. Thanks for the help!
